src/AdminConsole/Components/Shared/Credentials.razor.cs

```diff
@@ -1,52 +1,83 @@
-using Passwordless.AdminConsole.Services.AuthenticatorData;
+using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNetCore.Components;
+using Passwordless.AdminConsole.Helpers;
 using Passwordless.Common.Extensions;
+using Passwordless.Common.Validation;
 
-namespace Passwordless.AdminConsole.Pages.Shared;
+namespace Passwordless.AdminConsole.Components.Shared;
 
-public sealed class CredentialsModel
+public partial class Credentials : ComponentBase
```

Comment:
If you split the `@code` section into a separate file from `Credentials.razor` it would be named `Credentials.razor.cs`. But you have to make the class partial: https://learn.microsoft.com/en-us/aspnet/core/blazor/components/?view=aspnetcore-8.0#partial-class-support

Reply:
OK